docs/config-full-sample.yaml

```diff
@@ -211,6 +211,10 @@ sample:
     publishing:
       period: 30s
 
+    # Parameters for certificates generator
+    certificates:
+      resources: "/var/lib/disciplina/certificate-templates"
```

Comment:
Although, that's a pity that now one can't simply run the node using this config.